app/renderer.js

```diff
@@ -20,7 +20,7 @@ function displayMedia(imageScores, videoScores) {
     imgList.innerHTML = '';
 
     const items = [];
-    imageScores.forEach(function (item, index) {
+    imageScores.forEach(function (item) {
         const itemLoc = getImageLocation(item[0]);
```

Comment:
this `item[]` kinda pattern will cause mistakes and changes very hard in future. How about assign all in meaningful variables in the beginning and then use later?

Reply:
makes sense